src/silx/app/view/main.py

```diff
@@ -44,6 +44,14 @@ def createParser():
         'files',
         nargs=argparse.ZERO_OR_MORE,
         help='Data file to show (h5 file, edf files, spec files)')
+    parser.add_argument(
+        '--slice',
+        dest='slices',
+        default=tuple(),
+        type=int,
+        nargs='+',
+        help='List of slice indices to open (Only for dataset)',
+    )
```

Comment:
I think it would be better to use a real slice here

```
--slice 1,2,3
```
instead of
```
--slice 1 2 3
```

As we already have the parser, it's costless

Reply:
`--slice 1 2 3` opens not just one slice (1,2,3) as a dataset but 3 slices.
Maybe best to call it `--slices`.

I haven't done it, but we could also support `--slice 1::4,:10,...,1 3:5,:,1` if needed.... but the current main need is to open slices in a 3D stack.